rmw_zenoh_cpp/src/detail/rmw_subscription_data.cpp

```diff
@@ -201,14 +163,15 @@ SubscriptionData::SubscriptionData(
 // enable_shared_from_this, which is not available in constructors.
 bool SubscriptionData::init()
 {
-  // TODO(Yadunund): Instead of passing a rawptr, rely on capturing weak_ptr<SubscriptionData>
-  // in the closure callback once we switch to zenoh-cpp.
-  z_owned_closure_sample_t callback;
-  z_closure(&callback, sub_data_handler, nullptr, this);
-
-  std::string topic_keyexpr = entity_->topic_info()->topic_keyexpr_;
-  z_view_keyexpr_t sub_ke;
-  if (z_view_keyexpr_from_str(&sub_ke, topic_keyexpr.c_str()) != Z_OK) {
+  if (entity_->topic_info()->qos_.reliability == RMW_QOS_POLICY_RELIABILITY_RELIABLE) {
+    RMW_ZENOH_LOG_WARN_NAMED(
+      "rmw_zenoh_cpp",
+      "`reliability` no longer supported on subscriber. Ignoring...");
+  }
```

Comment:
It has been removed from the rolling since the 1.0 porting. We've decided that this information is annoying and only useful for the developers.

Reply:
https://github.com/ros2/rmw_zenoh/pull/327/commits/8a419851a3b0ef96bfd54d69f51f30f2b1d0b4c7